components/validation/src/main/java/org/wso2/carbon/identity/x509Certificate/validation/constant/X509CertificateValidationConstants.java

```diff
@@ -16,7 +16,7 @@
  * under the License.
  */
 
-package org.wso2.carbon.identity.x509Certificate.validation;
+package org.wso2.carbon.identity.x509Certificate.validation.constant;
```

Comment:
Is this change must have? Better to keep the contract of the public references intact as much as possible.

Reply:
fixed with https://github.com/wso2-extensions/identity-x509-commons/pull/56/commits/ee5b98f65b697623ef65f7d1a6ee245e1c9f982d